api/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:16
+
+COPY . /apps/auth/api
```

Comment:
Can we install it on `/app` instead of `/apps/auth` Some other services stores the app on `/app` and I'd like to standardize it. Most container should just have 1 service installed under `/app` (if we do that, we can use `/apps/<name>` for that special occasion) .

Reply:
I agree the container should have just one app- the bindings from other projects are only regarding to config files e.g. warehouse reading `/apps/auth/api/config/auth.pub` (from the auth project). They do not execute other services besides the main one in the container.